server/controllers/stock/reports/index.js

```diff
@@ -98,14 +98,16 @@ function stockExitPatientReceipt(req, res, next) {
       u.display_name AS user_display_name, p.display_name AS patient_display_name,
       CONCAT_WS('.', '${identifiers.DOCUMENT.key}', m.reference) AS document_reference,
       CONCAT_WS('.', '${identifiers.PATIENT.key}', proj.abbr, p.reference) AS patient_reference, p.hospital_no,
-      l.label, l.expiration_date, d.text AS depot_name
+      l.label, l.expiration_date, d.text AS depot_name,
+      dm.text AS documentReference
```

Comment:
At some point we should decide as a team how to name identifiers in SQL. Like here we have ```AS depot_name``` and ```AS documentReference``` very confusing.

Reply:
You're right, it is confusing here because we already has `document_reference` on the line 99.